doc/contributing/collaborator-guide.md

```diff
@@ -540,6 +540,9 @@ The TSC serves as the final arbiter where required.
    landing. If you are unsure exactly how to format the commit messages, use
    the commit log as a reference. See [this commit][commit-example] as an
    example.
+6. Ideally check the recent commits on `main` to evaluate any potentially
+   risk or breakage that may happen, e.g. when two or more PRs both modified
+   the same subsystem or related subsystem(s).
```

Comment:
```suggestion
6. Ideally, check the recent commits on `main` to evaluate any potential
   risk or breakage that may happen, e.g., due to multiple PRs that modify
   the same subsystem or related subsystem(s).
```

Reply:
(Feel free to ignore if there is no intention to pursue this PR.)

Reply:
Better wording is always appreciated 👍